less/lists.less

```diff
@@ -394,15 +394,24 @@
   }
 }
 
-// + 1 more card per row than usual, max 4
+// + 1 more card per row than usual, max 7
```

Comment:
Is this comment still relevant? I mean that's no longer "1 more card per row", that could be "4 more cards"?